### PR TITLE
Pull the latest wpt once a minute

### PR DESCRIPTION
**Pull request by bmac**

### CODE REVIEW

File: playbook.yml [hunk 1]
```diff
@@ -76,6 +76,12 @@
     cron:
       name: Synchronize WPT
       job: "cd /root/web-platform-tests/ && git pull origin master && systemctl restart wpt"
+      minute: '*'
+
+  - name: Add cron job to restart wpt once a day to get any changes to wptserve
```
Comment (jugglinmike) [inline playbook.yml]: This obviates the other cron job, doesn't it?

Comment (bmac) [reply playbook.yml]: Oops I forgot to remove the restart from the other cron job.

**Comment by bmac** [comment]
@jugglinmike I've updated this pull request. There are now 2 cron jobs. 1 runs every minute and pulls the latest changes from git. The other runs once a day and restarts the service to get any changes in the wpt directory. I didn't want to restart the service every minute because restarting causes a momentary bit of downtime where there is nothing to respond to incoming http requests.

**Comment by jugglinmike** [comment]
For those cases that do involve a change to the server logic, I'm concerned that the heuristic would cause confusion. Contributors could prove that the server had the latest code, but its behavior wouldn't match up.

What do you think about a Bash script that compares the HEAD before and after pulling and only restarts in response to change?

**Comment by bmac** [comment]
That seems reasonable.

**Comment by bmac** [comment]
I've updated this pr to only restart the server when the sha of the HEAD changes.

**Comment by jugglinmike** [comment]
Fantastic!

**Pull request merged**